.github/GETTING_STARTED.md

```diff
@@ -0,0 +1,119 @@
+# Getting started
+
+Contributing is easy:
+
+1. [Fork](https://help.github.com/articles/fork-a-repo/) the repository
+2. Create a new branch for the feature/fix and give it an appropriate name, e.g. `my_fabulous_fix`
+3. Do your best work
+4. Build/Test as per the below
+5. Create a pull request back to the [appropriate branch](CONTRIBUTING.md#Merge-Process) (or to `master` if you're not sure)
+
+## Prerequisites
+
+- Visual Studio 2017 / Visual Studio for Mac 7 / JetBrains Rider 2019.1 or newer
+- Xamarin SDK (Visual Studio Workload: Mobile Development with .NET)
+- .NET Core SDK 3.1.100 or newer (or Visual Studio Workload: .NET Desktop Development -> .NET Core 3.1 development tools)
+
+For more information, please refer to the Xamarin.Forms requirements.
+https://docs.microsoft.com/en-us/xamarin/xamarin-forms/get-started/installation
+
+.NET Core SDK 3.1.100 or newer is required for building .NET Core 3.1 projects. (e.g. Fabulous.XamarinForms generator)
+
+## Understanding the structure
+
+There is 4 distinct libraries in this repository:
+- Fabulous
+- Fabulous.CodeGen
+- Fabulous.XamarinForms
+- Fabulous.StaticView.XamarinForms
+
+They are separated into different folders, except Fabulous which is at the root of the repository since it's its repository.  
+
+Each library follows the same base structure:
+
+| Name  | Description                                  |
+--------|----------------------------------------------|
+| samples | (If applicable) The sample projects showcasing how the library usage |
+| src   | The source code, split into 1 or more projects |
+| tests | The unit & integration test projects |
+
+Since we're hosting the documentation on GitHub Pages, the markdown files can be found in the `docs` folder at the root of the repository, each library has its own subfolder.
+
+As a general rule, each `fsproj` in the `src` folders will be packaged in a NuGet package.
+
+A `sln` file is available at the root of the repository.
+
+### The special case of Fabulous.XamarinForms
+
+To start, it has a few other folders in addition to `samples`, `src`, and `tests`.
+
+| Name  | Description                                  |
+--------|----------------------------------------------|
+| extensions | The source code of the bindings libraries of 3rd party XF libs (e.g. Maps, SkiaSharp) |
+| templates | The files used to create the `dotnet` templates (e.g. `fabulous-xf-app`) |
+| tools | The tools used to create Fabulous.XamarinForms. Most notably the bindings generator. |
+
+
+Then, since Xamarin.Forms has tens of controls with each tens of properties/events, with a new release each month or so, it is really difficult to follow and write the appropriate XF bindings manually to make it compatible with Fabulous.  
```

Comment:
maybe `tons` instead of `tens`?

Reply:
`dozens` is better for what I intended to say. :)